### PR TITLE
lyra, bazel 関連の記載を削除する

### PR DESCRIPTION
**Pull request by miosakuma**
sora-cpp-sdk をlyra, bazel で検索してヒットした箇所を削りました。
問題ないか確認していただけますか。

---
This pull request includes changes to the GitHub Actions workflow and the Visual Studio Code C++ properties configuration file, as well as a version update. The most important changes involve the removal of Bazel caching steps from the GitHub Actions workflow and the removal of the path to the third-party library `lyra` from the Visual Studio Code C++ properties configuration file.

GitHub Actions workflow changes:

* [`.github/workflows/build.yml`](diffhunk://#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L28-L37): Removed Bazel caching steps from the workflow. These steps were removed from three different jobs, which previously cached Bazel build outputs to speed up subsequent builds. The removal of these steps may impact build times. [[1]](diffhunk://#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L28-L37) [[2]](diffhunk://#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L104-L108) [[3]](diffhunk://#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L145-L149)

Visual Studio Code C++ properties configuration changes:

* [`.vscode/c_cpp_properties.json`](diffhunk://#diff-8f4e8cf66ff6479868eeeb084ef19fbb8bea6102cf86f8adec2180dcf38dc47dL19): Removed the path to the third-party library `lyra` from the include directories. This change was made in three different configurations, which may affect the ability of the C++ IntelliSense feature in Visual Studio Code to locate the `lyra` library's header files. [[1]](diffhunk://#diff-8f4e8cf66ff6479868eeeb084ef19fbb8bea6102cf86f8adec2180dcf38dc47dL19) [[2]](diffhunk://#diff-8f4e8cf66ff6479868eeeb084ef19fbb8bea6102cf86f8adec2180dcf38dc47dL138) [[3]](diffhunk://#diff-8f4e8cf66ff6479868eeeb084ef19fbb8bea6102cf86f8adec2180dcf38dc47dL165)

Version update:

* [`VERSION`](diffhunk://#diff-7b60b8e351cbb80c47459ffe2c79f1a26404871f49294780fe47ad0e58c09350L5): Removed the line specifying the Bazel version. This change suggests that Bazel is no longer being used in this project.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -101,11 +91,6 @@ jobs:
         run: |
           echo "user=`users`" >> $GITHUB_OUTPUT
         id: env
-      - name: Bazel cache
-        uses: actions/cache@v4
-        with:
-          path: "/private/var/tmp/_bazel_${{ steps.env.outputs.user }}"
-          key: bazel-${{ matrix.name }}-v2
       - run: pip3 install numpy setuptools
```
Comment (melpon) [inline .github/workflows/build.yml:94]: numpy も消して大丈夫なはず。setuptools は分からないです。

Comment (enm10k) [reply .github/workflows/build.yml:94]: setuptools も Lyra のビルド・エラー解消のために追加したはずなので、消して良いと思います

Comment (torikizi) [reply .github/workflows/build.yml:94]: bazel 関連のビルドが通らない時に追加した記憶なので、一緒に削除して大丈夫です。
https://github.com/shiguredo/sora-cpp-sdk/commit/6009017c463e4f14b177d163cc9577007e09021a

Comment (enm10k) [reply .github/workflows/build.yml:94]: https://github.com/shiguredo/sora-cpp-sdk/compare/02d38f2059aa...7c35747a1a10

Comment (miosakuma) [reply .github/workflows/build.yml]: 確認ありがとうございます。 9e70961 で numpy と setuptools を削除しました

**Review by melpon — APPROVED** [review]
よさそう

**Comment by miosakuma** [comment]
確認ありがとうございます、マージします。

**Pull request merged**